daemon/volumes.go

```diff
@@ -107,23 +117,42 @@ func parseVolumesFrom(spec string) (string, string, error) {
 	return id, mode, nil
 }
 
+// read-write modes
+var rwModes = map[string]bool{
+	"rw":   true,
+	"rw,Z": true,
+	"rw,z": true,
+	"z,rw": true,
+	"Z,rw": true,
+	"Z":    true,
+	"z":    true,
+}
+
+// read-only modes
+var roModes = map[string]bool{
+	"ro":   true,
+	"ro,Z": true,
+	"ro,z": true,
+	"z,ro": true,
+	"Z,ro": true,
+}
+
 func validMountMode(mode string) bool {
-	validModes := map[string]bool{
-		"rw": true,
-		"ro": true,
-	}
-	return validModes[mode]
+	return roModes[mode] || rwModes[mode]
 }
 
 func (container *Container) specialMounts() []execdriver.Mount {
```

Comment:
These aren't being used anywhere and looks like it should have been removed (or just renamed) in the volumes refactor as `networkMounts` has now replaced it.